core/src/main/java/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitor.java

```diff
@@ -161,6 +161,10 @@ public boolean hasMoreRecentlyUsedToken(@Nonnull User user, @Nullable ApiTokenPr
     @RequirePOST
     public HttpResponse doRevokeAllSelected(@JsonBody RevokeAllSelectedModel content) throws IOException {
         for (RevokeAllSelectedUserAndUuid value : content.values) {
+            if(value.userId == null){
```

Comment:
add space between if and ( and the same between ) and {

Reply:
1680x `if(` and 4782x `if (`. There is no real style guide in the project. I do the change to be consistent within the file.